TTHAnalysis/python/plotter/mcPlots.py

```diff
@@ -988,9 +989,11 @@ def printOnePlot(self,mca,pspec,pmapIn,mytotal=None,makeCanvas=True,outputDir=No
                 signorm = None; datnorm = None; sfitnorm = None
                 if options.showSigShape or options.showIndivSigShapes or options.showIndivSigs: 
                     signorms = doStackSignalNorm(pspec,pmap,options.showIndivSigShapes or options.showIndivSigs,extrascale=options.signalPlotScale, norm=not options.showIndivSigs)
+                    #print(signorms)
                     for signorm in signorms:
+                        #print(outputDir)
                         if outputDir: 
-                            signorm.SetDirectory(outputDir); outputDir.WriteTObject(signorm)
+                            signorm.SetDirectory(outputDir); #outputDir.WriteObjectAny(signorm)
```

Comment:
After all, do we need to comment this out?
Also, since this is a shared file (in the plotter directory), I would remove the print out lines.